### PR TITLE
update file and directory structure in docs

### PR DESCRIPTION
**Pull request by melsyt**
Fixes https://github.com/pypa/warehouse/issues/4969

### CODE REVIEW

File: docs/application.rst [hunk 1]
```diff
@@ -86,22 +86,25 @@ Directories within the repository:
...
 
-  - `legacy/ <https://github.com/pypa/warehouse/tree/master/warehouse/legacy>`_ - most of the read-only APIs implemented here
-  - `forklift/ <https://github.com/pypa/warehouse/tree/master/warehouse/forklift>`_ - :ref:`upload-api-forklift`
   - `accounts/ <https://github.com/pypa/warehouse/tree/master/warehouse/accounts>`_ - user accounts
   - `admin/ <https://github.com/pypa/warehouse/tree/master/warehouse/admin>`_ - application-administrator-specific
   - `cache/ <https://github.com/pypa/warehouse/tree/master/warehouse/cache>`_ - caching
   - `classifiers/ <https://github.com/pypa/warehouse/tree/master/warehouse/classifiers>`_ - frame trove classifiers
   - `cli/ <https://github.com/pypa/warehouse/tree/master/warehouse/cli>`_ - entry scripts and
     `the interactive shell <https://warehouse.readthedocs.io/development/getting-started/#running-the-interactive-shell>`_
+  - `email/ <https://github.com/pypa/warehouse/tree/master/warehouse/email>`_ - sends emails
```
Comment (di) [inline docs/application.rst]:
```suggestion
  - `email/ <https://github.com/pypa/warehouse/tree/master/warehouse/email>`_ - services for sending emails
```

File: docs/application.rst [hunk 2]
```diff
@@ -86,22 +86,25 @@ Directories within the repository:
...
+  - `forklift/ <https://github.com/pypa/warehouse/tree/master/warehouse/forklift>`_ - :ref:`upload-api-forklift`
   - `i18n/ <https://github.com/pypa/warehouse/tree/master/warehouse/i18n>`_ - internationalization
+  - `legacy/ <https://github.com/pypa/warehouse/tree/master/warehouse/legacy>`_ - most of the read-only APIs implemented here
   - `locales/ <https://github.com/pypa/warehouse/tree/master/warehouse/locales>`_ - internationalization
   - `manage/ <https://github.com/pypa/warehouse/tree/master/warehouse/manage>`_ - logged-in user functionality (i.e., manage account &
     owned/maintained projects)
+  - `metrics/ <https://github.com/pypa/warehouse/tree/master/warehouse/metrics>`_ - registers metric service
```
Comment (di) [inline docs/application.rst]:
```suggestion
  - `metrics/ <https://github.com/pypa/warehouse/tree/master/warehouse/metrics>`_ - services for recording metrics
```

File: docs/application.rst [hunk 3]
```diff
@@ -86,22 +86,25 @@ Directories within the repository:
...
   - `migrations/ <https://github.com/pypa/warehouse/tree/master/warehouse/migrations>`_ - changes to the database schema
   - `packaging/ <https://github.com/pypa/warehouse/tree/master/warehouse/packaging>`_ - models
   - `rate_limiting/ <https://github.com/pypa/warehouse/tree/master/warehouse/rate_limiting>`_ - rate limiting to prevent abuse
   - `rss/ <https://github.com/pypa/warehouse/tree/master/warehouse/rss>`_ - RSS feeds: :doc:`api-reference/feeds`
+  - `search/ <https://github.com/pypa/warehouse/tree/master/warehouse/search>`_
```
Comment (di) [inline docs/application.rst]:
```suggestion
  - `search/ <https://github.com/pypa/warehouse/tree/master/warehouse/search>`_ - utilities for building and querying the search index
```

File: docs/application.rst (hunk 1 above)
Comment (melsyt) [reply docs/application.rst]: updated

File: docs/application.rst (hunk 3 above)
Comment (melsyt) [reply docs/application.rst]: updated

File: docs/application.rst (hunk 2 above)
Comment (melsyt) [reply docs/application.rst]: updated

**Comment by melsyt** [comment]
@brainwane Sure, what should I write for the description?

**Comment by brainwane** [comment]
@melsyt I suggest:

> Jinja templates for web pages, emails, etc.

**Comment by di** [comment]
Thanks @melsyt!

**Pull request merged**